scripts/util/keystore-cert-import

```diff
@@ -138,6 +141,6 @@ if [ -f "$SYSTEM_FILE" ]; then
 fi
 
 # Fix file permission
-chmod 600 $CACERT_FILE
-chmod 600 $CERT_FILE
-chmod 600 $PRIVKEY_FILE
+chmod 750 $CACERT_FILE
+chmod 750 $CERT_FILE
+chmod 750 $PRIVKEY_FILE
```

Comment:
are execute bits needed for these files? seems 640 would be good.

Reply:
These 3 are included on the command above so at this stage the permissions are overriden

Reply:
same as Daan - I don't think we need executable flag on certificate/keys file; read should be good?